modules/administration-guide/partials/con_controller-requirements.adoc

```diff
@@ -39,7 +39,7 @@ The Workspace Controller consists of a set of five services running in five dist
 |16 MiB
 |===
 
-These default values are sufficient when the {prod-short} Workspace Controller manages a small amount of {prod-short} workspaces. For larger deployments, increase the memory limit. See the xref:installation-guide:advanced-configuration-options-for-the-che-server-component.adoc[] article for instructions on how to override the default requests and limits. For example, the hosted version of {prod-short} that runs on link:https://che.openshift.io[] uses 1 GB of memory.
+These default values are sufficient when the {prod-short} Workspace Controller manages a small amount of {prod-short} workspaces. For larger deployments, increase the memory limit. See the xref:installation-guide:advanced-configuration-options-for-the-che-server-component.adoc[] article for instructions on how to override the default requests and limits. For example, the Eclipse Che hosted by Red Hat that runs on link:https://workspaces.openshift.com[] uses 1 GB of memory.
```

Comment:
Well, isn't it CodeReadyWorkspaces?

Reply:
it is, but the official naming for now is Eclipse Che hosted by Red Hat - https://www.eclipse.org/che/docs/che-7/hosted-che/hosted-che/#_differences_between_eclipse_che_and_codeready_workspaces

It is misleading, but there was a decision for now to leave the naming as-is (even though I do agree it is a bit misleading)